src/setup/rgw.py

```diff
@@ -167,7 +167,7 @@ def start(conf: MappedConf, index: str):
 
         Log.info('Starting radosgw service.')
         log_path = Rgw._get_log_dir_path(conf)
-        log_file = os.path.join(log_path, f'{COMPONENT_NAME}-{index}')
+        log_file = os.path.join(log_path, f'{COMPONENT_NAME}-{index}.log')
```

Comment:
earlier seperate log file was getting generated while starting the rgw service (rgw-1 & rgw-1.log)
dumping the logs to the same log file. now only one log file will be created i.e rgw-1.log

Reply:
this is for the rgw service itself right? @gargpalak048

Reply:
> Or one log file was for cortx rgw startup code and second log file then for actual rgw service ?
yes @cdeshmukh 
rgw-1 is the console log generated while starting rgw service.
rgw-1.log is actual rgw log file.

Do we need to maintain two separate log files?

Reply:
Addressed.
Keeping two log files:
rgw_startup.log & rgw-{instance}.log